node/src/cli.rs

```diff
@@ -14,6 +14,25 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with Zeitgeist. If not, see <https://www.gnu.org/licenses/>.
+//
+// This file incorporates work covered by the following copyright and
+// permission notice:
+//
+//     Copyright 2017-2020 Parity Technologies (UK) Ltd.
+//     This file is part of Polkadot.
+//
+//     Polkadot is free software: you can redistribute it and/or modify
+//     it under the terms of the GNU General Public License as published by
+//     the Free Software Foundation, either version 3 of the License, or
+//     (at your option) any later version.
+//
+//     Polkadot is distributed in the hope that it will be useful,
+//     but WITHOUT ANY WARRANTY; without even the implied warranty of
+//     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//     GNU General Public License for more details.
+//
+//     You should have received a copy of the GNU General Public License
+//     along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
It's the same license as we use, we should just put their Copyright statement "Copyright 2017-2020 Parity Technologies (UK) Ltd." next to ours.

Reply:
Sorry, I was 100% I had removed that. Is fixed now.